airflow/providers/amazon/aws/utils/s3.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import numpy
+import pandas
+
+
+def fix_int_dtypes(df: pandas.DataFrame) -> None:
```

Comment:
Could you add comment to describe this function? The implementation of this function is not really clear and some comments would help

Reply:
Thanks for your comment. I am using existed this function from SqlToS3Operator._fix_int_dtypes.

Reply:
I know and I know it's not your fault but the potential future developers who will use/modify this function, I think some comments would be good :)